### PR TITLE
Feature/traceback concat

### PR DESCRIPTION
**Pull request by buehlere**
- overhauling `pv maf concat` to match new file - operation standards
- tagging for traceback files
- contribution doc
- PR template
- add code owners
- clean-up

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
+# Contributing to Post-Processing
```
Comment (rnaidu) [inline CONTRIBUTING.md:1]: Hi Eric, I followed these commands when working on the remove_variants_by_annotations.py conversion to a subcommand and was able to understand the instructions clearly. Great job on explaining the steps/resources needed to set up these commands!

File: README.md [hunk 2]
```diff
@@ -2,58 +2,6 @@
 
 This hosts multiple scripts necessary for filtering and processing of variant calls in the vcfs/txt file generated by callers.
 
-## Callers Supported
-`pv` is the main command for the `postprocessing_variant_calls` package see `pv --help` to see supported variant callers commands. 
-
-### VarDictJava
-
-The sub-command `pv vardict` allows users to perform post-processing on VarDictJava output. The two supported inputs to `pv vardict` from VarDictJava are `single` and `case-control` vcfs.
```
Comment (rnaidu) [inline README.md:10]: very minor change, but I'd recommend adding the version of VarDictJava in the README. mostly due to the fact that the docker image created for VarDict that is compatible with M1 is version 1.8.2 (same version used by MSK), but the version used by nf-core is 1.8.3

File: README.md [hunk 3]
```diff
@@ -2,58 +2,6 @@
...
-
-`pv vardict single filter --inputVcf data/Myeloid200-1.vcf  --tsampleName Myeloid200-1  -ad 1 -o data/single`
-
-There are various options and input specifications for filtering so see `pv vardict single filter --help` or `pv vardict single case-sontrol --help` for help. 
-
-See `example_calls.sh` for more example calls. 
-
-### Maf 
-
-maf concat examples:
```
Comment (rnaidu) [inline README.md:30]: again very minor, but maybe adding a sentence at the beginning of the README (similar to line 10) with a quick explanation to what the category of 'maf' commands are intended to do in post-processing. When the vcf2maf module has been approved in msk-modules, we could even add a hyperlink to it in the README here

File: data/method_process.py [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+        "-sep",
+        help="Specify a seperator for delimited data.",
+        callback=check_separator,
+    ),
+):
+    # prep maf
+    mafa = MAFFile(maf, separator)
+    mafa = mafa.tag("common_variant")
+    mafa.to_csv(f"{output_maf}".format(outputFile=output_maf), index=False, sep="\t")
+    return 0
```
Comment (rnaidu) [inline data/method_process.py:60]: Not a review comment but a question-- I noticed you return "0" at the end of your sub-command functions. What is the purpose of returning 0 as compared to a string value indicating the execution of the sub-command has completed or not returning anything at all?

**Review by rnaidu — APPROVED** [review]
Great job! I added a couple of recommendations for adding documentation in the contributing and README docs. The changes are minor and won't affect things downstream, so I am approving the review.

File: README.md (hunk 2 above)
Comment (buehlere) [reply README.md:10]: good call. I added the version in the `docs/VARDICT.md` and added a link in the README.

File: README.md (hunk 3 above)
Comment (buehlere) [reply README.md:30]: similar to above I added a link to the README that points to `docs/maf`

File: data/method_process.py (hunk 4 above)
Comment (buehlere) [reply data/method_process.py:60]: In python, if I'm writing a file in a function as opposed to returning something, I still like to return a 0 to indicate a successful run of the function. Some languages like C require you return something at the completion of a function. It's not required in python, but something I think is decent style. We can change the return as well.

**Pull request merged**